src/gui/mainwindow.cpp

```diff
@@ -1598,17 +1598,14 @@ void MainWindow::on_actionSearchWidget_triggered()
 
         bool res = false;
 
-        if ((pythonVersion == 2) || (pythonVersion == 3)) {
+        if (pythonVersion == 3) {
             // Check Python minimum requirement: 2.7.9 / 3.3.0
             QString version = Utils::Misc::pythonVersionComplete();
             QStringList splitted = version.split('.');
             if (splitted.size() > 2) {
                 int middleVer = splitted.at(1).toInt();
-                int lowerVer = splitted.at(2).toInt();
-                if (((pythonVersion == 2) && (middleVer < 7))
-                    || ((pythonVersion == 2) && (middleVer == 7) && (lowerVer < 9))
-                    || ((pythonVersion == 3) && (middleVer < 3))) {
-                    QMessageBox::information(this, tr("Old Python Interpreter"), tr("Your Python version (%1) is outdated. Please upgrade to latest version for search engines to work.\nMinimum requirement: 2.7.9 / 3.3.0.").arg(version));
+                if ((pythonVersion == 3) && (middleVer < 3)) {
```

Comment:
But then for a user with python2 there will be no message that python is outdated? If program reports that it can't find python when one is installed, that is confusing.